rclpy/rclpy/parameter.py

```diff
@@ -14,6 +14,7 @@
 
 from enum import Enum
 
+from rcl_interfaces.msg import Parameter as RCLParameter
```

Comment:
This is a strange rename to me, at first glance, since it's clearly a message a not a type defined within `rcl`. I see that it comes from `rcl_interfaces` which make some sense, but it was surprising to me. `ParameterMsg` is a clearer name to me, since it will always be a message but my not always be in the rcl interfaces or have anything to do with rcl in the future.

This is a pure nitpick, feel free to ignore it.

Reply:
:+1: I meant to scope that with the [comment with about renaming the method](https://github.com/ros2/rclpy/pull/226#discussion_r210959738) and apparently didn't post it. I'm also in favor of a name expressing the fact that this is a message and not a structure coming from rcl.

Following that logic, the method converting to and from parameter message could also be named to refer to messages rather than rcl_interfaces

Reply:
> This is a strange rename to me, at first glance

I agree with both of you given the explanation and have pushed 6ddfbba to use ParameterMsg and to/from_parameter_msg. As Mikael suspected the rcl prefix was being used because it was an rcl_interfaces type but using just rcl adds confusing with rcl types. ParameterMsg is apt and available so it's what we'll use.